README.md

```diff
@@ -37,6 +38,37 @@ Brion provides the following major features:
   [vmmlib](http://vmml.github.io/vmmlib),
   [Lunchbox](http://eyescale.github.io/Lunchbox-1.8/index.html),
   [Boost](http://www.boost.org/doc/libs).
+* Higher level classes:
+  * Read only morphology (brion::v2::Morphology)
+
+# Library versions {#Versions}
+
+Brion is evolving into a version 2 of its API. This new version will provide
+higher level classes to facilitate queries and manipulation of the data model
+provided by the low level IO objects. Version 2 is experimental at this
+moment, so it is not the default API and the major number of the library is
+still 1.
+
+## Version 1 compatibility
+
+If you are using the Brion 1 you don't need to do anything to compile your
+code against the latest version of Brion. However, the last release of
+version 1 is not binary compatible with the version 2 library, so
+you will need to recompile is you want to take advantage of the latest
+features that are backwards compatible.
+
+## Using version 2
+
+In order to use Brion 2 there is nothing special to be done to compile Brion
+itself. In client code you have to define the macro \c BRION_USE_V2_API to
+enable this version. When this macro is set, all the new high level classes
+will live in the namespace \c brion and classes which clash with new ones are
+moved into the \c brion::io If the macro is not enabled, version 2 classes
+will still be available inside the namespace \c brion::v2.
+
+There is no need to recompile Brion to switch between versions. It is
+guaranteed that this macro only changes the namespace layout and that class
+interfaces remain equal.
```

Comment:
Right and right.
You mean that io is redundant right now or in the future? If io is going to be in v2 it must be also now so you can already write v2 code.

Sure, this is my proposal. If you have a better one, your welcome. The main question is the following: which are the first class citizen classes in v2, the low level ones or the new ones.